src/openrct2/GameState.cpp

```diff
@@ -32,6 +32,9 @@
 
 using namespace OpenRCT2;
 
+// speed steps in percent. 25% is 1/4 of normal speed (100%).  
+static uint32_t GameSpeeds[GAMESPEED_HYPER+1] = { 0, 25, 100, 200, 400, 1000, 10000 };
```

Comment:
Add a trailing comma after `10000` please

Reply:
Is this code really valid? The enum you use here has some oddities like not being contiguous and being assigned seemingly arbitrary values.